Projects/Chat/Sources/Views/ChatScreenViewModel.swift

```diff
@@ -46,23 +46,23 @@ public class ChatScreenViewModel: ObservableObject {
             AskForRating().askAccordingToTheNumberOfSessions()
         }
         log.addUserAction(type: .click, name: "Chat open", error: nil, attributes: nil)
-        let store: ChatStore = globalPresentableStoreContainer.get()
-        storeActionSignal = store.actionSignal.publisher
-            .sink(receiveValue: { [weak self] action in guard let self = self else { return }
-                if case let .navigation(navigationAction) = action {
-                    if case let .linkClicked(url) = navigationAction {
-                        if let deepLink = DeepLink.getType(from: url), deepLink == .helpCenter {
-                            log.addUserAction(
-                                type: .custom,
-                                name: "Help center opened from the chat",
-                                error: nil,
-                                attributes: ["haveSentAMessage": self.haveSentAMessage]
-                            )
-                        }
-                    }
-                }
-            }
-            )
+        //        let store: ChatStore = globalPresentableStoreContainer.get()
```

Comment:
@sladan-hedvig not sure how we should handle this

Reply:
Fixed